Contributing.md

```diff
@@ -0,0 +1,50 @@
+# Stakeholders
+
+- **Project Manager**: responsible for aligning task creation with broader program direction through discussion with task manager(s). May comment on tasks.
```

Comment:
I think the most important task for the project manager is to ensure that the project members are focused. So ensure there is a focus for which there is a consensus, and then ensure that the project members stick to it. Suggested rephrase:
responsible for ensuring that the project has a focus that the project members agree on, and make sure that everyone in the project sticks to that focus.

Reply:
agreed. good rephrase, it captures the role better.